scripts/errorCheck.ts

```diff
@@ -1,6 +1,6 @@
-const fs = require("fs");
-const async = require("async");
-const glob = require("glob");

```

Comment:
These are arguably more correct though, right?

Reply:
Do you mean the old requires, or the new imports? (Some of the scripts already used imports; I have these changes also on my module transform branch too.)

Reply:
(The output code is the same; with the imports, we generate `const fs = require("fs")` exactly)

Reply:
I meant the old `require`s since those correspond to these being true CommonJS require calls anyway. It probably doesn't matter, so long as `glob` doesn't get called.

Reply:
Ah, are you suggesting `import glob from "glob"`? I can do that, so long as we enable `esModuleInterop`. I opted to not change the tsconfig settings since `import *` seem to work.

Reply:
Right so that's the thing - `import`/`require` allows us to not worry about interop at all because we get to just say "yeah, the thing Node actually does with CJS.

The alternative is to swap to ESM scripts which I am actually open to as well for dog-fooding purposes.

Reply:
Yes, that is actually the next step I wanted to take, but I wanted to get them to type check first.